itou/templates/job_seekers_views/list.html

```diff
@@ -46,4 +46,39 @@ <h1 class="m-0">Candidats</h1>
 {% block script %}
     {{ block.super }}
     <script src='{% static "js/htmx_compat.js" %}'></script>
+    <script nonce="{{ CSP_NONCE }}">
+        const breakpointMD = getComputedStyle(document.documentElement).getPropertyValue('--bs-breakpoint-md');
+        if (window.matchMedia('(min-width: ' + breakpointMD + ')').matches && document.querySelector('#introJsBtnPostuler')) {
+            introJs().setOptions({
+                    disableInteraction: true,
+                    showProgress: false,
+                    showBullets: false,
+                    skipLabel: '',
+                    nextLabel: 'Suivant',
+                    prevLabel: 'Précédent',
+                    doneLabel: 'Fin',
+                    steps: [{
+                        title: 'Postulez directement pour ce candidat !',
+                        element: document.querySelector('#introJsBtnPostuler'),
```

Comment:
Un peu plus clair et plus rapide.
```suggestion
                        element: document.getElementById('introJsBtnPostuler'),
```